src/main/kotlin/racingcar/DriveRule.kt

```diff
@@ -0,0 +1,17 @@
+package racingcar
+
+import java.util.concurrent.ThreadLocalRandom
+
+class DriveRule : DriveRangeStrategy {
+    companion object {
+        private const val FORWARD_NUMBER = 4
+
+        fun random(): Int {
+            return ThreadLocalRandom.current().nextInt(0, 9)
+        }
+    }
+
+    override fun isForward(number: Int): Boolean {
+        return number >= FORWARD_NUMBER
+    }
```

Comment:
DriveRangeStrategy에 인터페이스를 한이유는 추상화이기도 합니다.
4이상만 앞으로 움직일수 있다는 전략을 내부에서만 가지고 있는게 어떨까요?

```suggestion
    override fun isForward(number: Int): Boolean {
        return random() >= FORWARD_NUMBER
    }
```